chapters/03-implementation.Rmd

```diff
@@ -97,11 +97,13 @@ A critical insight is that the Gaussian kernel density estimator $\widehat{f}_{h
 
```

Comment:
[9] Line 94: yeah, that maybe true that it's difficult (for you and me), but it doesn't matter (don't judge). Does the book really add something? Mentioning it?

Reply:
Since it is indeed difficult, can you give a qualitative overview of the _idea_? It gets quite mathematical afterwards